### PR TITLE
pass template argument as intended

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: bin/VMSlogger [hunk 1]
```diff
@@ -87,15 +87,16 @@ parser.add_argument(
 parser.add_argument(
     "--template",
     action="store",
+    dest="template",
     default="${name}_%Y-%m-%dT%H:%M:%S.${ext}",
     type=str,
-    help="template used to construct filename",
+    help="template used to construct filename. Defaults to ${name}_%Y-%m-%dT%H:%M:%S.${ext}",
```
Comment (teweitsai) [inline bin/VMSlogger:93]: Defaults to? or Default to?

File: bin/VMSlogger [hunk 2]
```diff
@@ -87,15 +87,16 @@ parser.add_argument(
...
 )
 parser.add_argument(
     "--workdir",
     action="store",
     dest="workdir",
-    default="",
+    default=None,
     help="directory where files will be stored",
```
Comment (teweitsai) [inline bin/VMSlogger:100]: You could write the default is `None` in `help`.

**Review by teweitsai — APPROVED** [review]
Looks good to me. Please consider to improve the document as in the comments. Thanks!

**Comment by pkubanek** [comment]
Changed as requested - thanks for feedback.

**Pull request merged**